### PR TITLE
fix: sharing link from Easel NFT image link is not valid

### PR DESCRIPTION
**Pull request by kf-rns**
## Description
When sharing link from Easel NFT image link is not valid as a text in Discord or Text message invalid image for NFT- As of now there is  2021 Team of the Year image with Fifa Team.

Fixes #
- Changed hard codded urls and made them dynamic from settings.json
- Changed API for getting receipes to get single recipe instead of all recipes

## Checklist
- [x] Targeted PR against correct branch.
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Run `meteor npm run lint`
- [ ] Added an entry to the `CHANGELOG.md` file.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

File: imports/startup/server/index.js [hunk 1]
```diff
@@ -55,36 +50,37 @@ Meteor.startup(() => {
...
             } catch (e) { 
                 console.log(e);
-            }    
-            for(i = 0; i < recipes.length; i++){
-                selectedRecipe = recipes[i];
-                if(selectedRecipe.ID == recipe_id){
-                    break;
-                }
             }
+            // for(i = 0; i < recipes.length; i++){
```
Comment (MikeSofaer) [inline imports/startup/server/index.js:68]: No commented out code please

File: imports/startup/server/index.js [hunk 2]
```diff
@@ -229,22 +225,22 @@ Meteor.startup(() => {
...
             } catch (e) { 
                 console.log(e);
             }    
-            for(i = 0; i < recipes.length; i++){
-                selectedRecipe = recipes[i];
-                if(selectedRecipe.ID == recipe_id){
-                    break;
-                }
-            }
+            // for(i = 0; i < recipes.length; i++){
```
Comment (MikeSofaer) [inline imports/startup/server/index.js:238]: please delete if not needed

**Review by MikeSofaer — APPROVED** [review]
please delete commented code

**Review by kjawadDeveloper2 — APPROVED** [review]
For now its fine but we will soon need to support the videos  and audio as there url doesn't have image.

**Pull request merged**